avidemux_de.ts

```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
 If you just copy the video without reencoding,
  you should keep them.
 Drop timing informations ?</source>
-        <translation type="unfinished"></translation>
+        <translation>Einige Timing-Informationen sind fehlerhaft.
+Dies kommt bei manchen Aufnahmeprogrammen (Capturing) vor.
+Wenn Sie das Video neukodieren möchen, sollten diese Informationen gelöscht werden,
+ andernfalls wird dies fehlende Bilder/ruckelndes Video verursachen.
+Wenn Sie das Video nur kopieren möchten, anstatt es neuzukodieren,
+ sollten Sie die Informationen beibehalten.
+Timing-Informationen löschen?</translation>
     </message>
 </context>
 <context>
     <name>CalculatorDialog</name>
     <message>
         <location filename="../ADM_userInterfaces/ADM_dialog/calculator.ui" line="+16"/>
         <source>Calculator</source>
-        <translation type="unfinished">Video-Kalkulator</translation>
+        <translation>Video-Kalkulator</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Target</source>
-        <translation type="unfinished">Format und Größe des Ausgabedatei</translation>
+        <translation>Format und Größe des Ausgabedatei</translation>
```

Comment:
If touching translations of dead code is questionable, touching them and not fixing an occurrence of a wrong article makes even less sense, I guess.

Reply:
Oops, you're right. Actually, I checked every translation before I set it to finished, but I must have overlooked it here.
I double-checked the changes and made a commit.